packages/odyssey-react-mui/src/Radio.tsx

```diff
@@ -114,17 +129,30 @@ const Radio = ({
         <MuiRadio
           inputProps={{
             "data-se": testId,
+            "aria-disabled": isReadOnly,
+            readOnly: isReadOnly,
+            tabIndex: isReadOnly ? 0 : undefined,
```

Comment:
Do we want this change?
```suggestion
            "aria-disabled": isDisabled || isReadOnly,
            readOnly: isReadOnly,
            tabIndex: isReadOnly ? 0 : undefined,
```

Reply:
good point. Makes sense to include on the disabled state as well. I found that it is required for read-only but I don't see a reason to not include it for `disabled` as well.